RSN 27, 29, 30, 31, 32, 33 Completion Notices (#423)
rapidsai/docs: _notices/rsn0030.md

# Update meta-data for notice
notice_id: 30 # should match notice number
notice_pin: true # set to true to pin to notice page
title: "Docker Image Changes in Release v23.08"
notice_author: RAPIDS TPM
notice_status: Completed
notice_status_color: green
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change
notice_rapids_version: "v23.08"
notice_created: 2023-04-20
# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2023-08-17
---

## Overview

> RAPIDS is making a number of changes to its end-user Docker images to improve functionality, performance, and maintainability. These changes are targeted to be completed in the RAPIDS 23.08 release.

Reviewer: Do we want to put this in past tense, to indicate that it has happened (rather than targeted for the future)?
```suggestion
The RAPIDS 23.08 release has made changes to end-user Docker images to improve functionality, performance, and maintainability.
```

While many of these changes should be seamless to users, **some of them are breaking**. Users should review the full list of changes in the associated issue: [https://github.com/rapidsai/docker/issues/539](https://github.com/rapidsai/docker/issues/539)

## Impact

> Users should be particularly aware of the breaking changes as noted above. In order to extend the transition time for impacted users, RAPIDS will continue to publish containers in the old namespace (i.e. rapidsai/rapidsai-core) for one additional release until 23.08.

Reviewer: I think the latter sentence is unclear, and implies that "one additional release" was made (in 23.08) with the old names, which isn't right. Let's remove it so there is no ambiguity.
```suggestion
Users should be particularly aware of the breaking changes as noted above.
```
